docs/README.md

```diff
@@ -13,8 +13,18 @@ This is the documentation of AMS, it's focused on the **0.10.x version.**
 ## How to
 
 - [How to add root key](howto/add_root_key.md)
-- [How to add pagination links](howto/add_pagination_links.md)
 - [Using AMS Outside Of Controllers](howto/outside_controller_use.md)
+- [How to add pagination links](howto/add_pagination_links.md)
+
+#### JSON API Resource `type`
+
+##### Implicit
+
+Resource class name is used, e.g. the type of a `Post` will be `posts` in the resulting json api resource.
```

Comment:
Maybe expand a bit, for instance with the generated `type` for `Namespace::MultiWordResource` or such.